test/hotspot/jtreg/serviceability/jvmti/ObjectMonitorUsage/ObjectMonitorUsage.java

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8247972
+ *
+ * @summary converted from VM Testbase nsk/jvmti/GetObjectMonitorUsage/objmonusage003
+ * DESCRIPTION
+ *     The test checks if the JVMTI function GetObjectMonitorUsage returns
+ *     the expected values for the owner, entry_count, water_count
+ *     fields of JVMTI_monitor_info.
+ *     The testcases are the following:
+ *       - unowned object without any waitings
+ *       - unowned object with waitings to be notified
+ *       - owned object without any waitings
+ *       - owned object with N waitings to enter the monitor
+ *       - owned object with N waitings to be notified
+ *       - owned object with N waitings to enter, from 0 to N waitings to re-enter,
+ *         from N to 0 waitings to be notified
```

Comment:
"waitings" is not grammatically valid. Suggestion: s/waitings/threads waiting/

Reply:
Thanks, fixed now.